Services/OperationService/OperationService.csproj

```diff
@@ -5,9 +5,13 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Remove="BotLogic\**" />

```

Comment:
Why you need this?

Reply:
I can't ask you (but i will ask others about it )